docs/sdks/client-sdks/flutter/assignments.mdx

```diff
@@ -0,0 +1,178 @@
+    return handleVersionA();
+  case 'version-b':
+    return handleVersionB();
+  default:
+    return handleDefaultVersion();
+}
+```
+
+## JSON Assignments
+
+JSON flags work best for advanced configuration use cases. The JSON flag can include structured information such as:
+
+- The text of marketing copy for a promotional campaign
+- The address of a different hero image
+
+Using this pattern, a team can make minor changes to the copy and design of a website without having to go through an entire code release process.
+
+```dart
+final defaultAssignment = {
+  'hero': false,
+  'heroImage': 'placeholder.png',
+  'heroTitle': 'Placeholder Hero Title',
+  'heroDescription': 'Placeholder Hero Description'
+};
+
+final campaignJson = Eppo.getJSONAssignment(
+  'flag-key-123',
+  defaultAssignment
+);
+
+if (campaignJson != null) {
+  campaign.hero = true;
+  campaign.heroImage = campaignJson['heroImage'] as String?;
+  campaign.heroTitle = campaignJson['heroTitle'] as String? ?? '';
+  campaign.heroDescription = campaignJson['heroDescription'] as String? ?? '';
+}
+
+// Use the campaign settings in your component
+renderHero(campaign.heroImage, campaign.heroTitle, campaign.heroDescription);
+```
+
+## Integer and Numeric Assignments
+
+Integer and numeric assignments work the same way but return either an integer or a floating point number. 
+These assignments are useful where you want to use a numeric value to drive business logic such as pricing on an item or a number of items to display in a list.
+
+```dart
+// Example of getting an integer assignment
+final numberOfItems = Eppo.getIntegerAssignment(
+  'flag-key-123',
+  0  // default value
+);
+
+// Example of getting a numeric assignment
+final price = Eppo.getNumericAssignment(
+  'flag-key-123',
+  0.0  // default value
+);
+
+// Use the assignments to drive business logic
+renderItemList(numberOfItems);
+renderPrice(price);
+```
+
+## Assignment Logger Schema
+
+The SDK will invoke the `logAssignment` method with an `AssignmentEvent` object containing the following fields:
+
+```dart
+class AssignmentEvent {
+  /// The allocation the subject was assigned to
+  final String? allocation;
+
+  /// The experiment identifier
+  final String? experiment;
+
+  /// The feature flag identifier
+  final String featureFlag;
+
+  /// The format of the assignment
+  final String format;
+
+  /// The variation key that was assigned
+  final String? variation;
+
+  /// The subject identifier
+  final String subject;
+
+  /// The timestamp of the assignment
+  final String timestamp;
+
+  /// The subject attributes
+  final Map<String, dynamic>? subjectAttributes;
+
+  /// Additional metadata
+  final Map<String, dynamic>? metaData;
+
+  /// Details about the evaluation
+  final Map<String, dynamic>? evaluationDetails;
+```
+
+### Logging data to your data warehouse
+
+Eppo's unique architecture makes it so Eppo never has access to your data. This means that you can use the assignment 
```

Comment:
> Eppo never has access to your data

This may make slightly less sense with precomputed assignments. Perhaps we say Eppo never _stores_ your data?